update.go

```diff
@@ -185,15 +188,24 @@ func checkUpdate(curProfile *Profile, all, force bool) error {
 func checkImageDate(profile *Profile, checkData ImageCheckData, force bool) []ImageDef {
 	var imageCandidates, images []ImageDef
 
-	// Dual arch profile
-	switch {
-	case profile.ImageAMD64 != "" && profile.ImageARM64 != "":
+	// multi arch profiles
+	if profile.ImageAMD64 != "" {
 		imageCandidates = append(imageCandidates, ImageDef{Image: profile.ImageAMD64, Platform: "linux/amd64"})
+	}
+	if profile.ImageARM64 != "" {
 		imageCandidates = append(imageCandidates, ImageDef{Image: profile.ImageARM64, Platform: "linux/arm64"})
-	case profile.Image != "":
+	}
+	if profile.ImageARM != "" {
+		imageCandidates = append(imageCandidates, ImageDef{Image: profile.ImageARM, Platform: "linux/arm"})
+	}
+	if profile.ImageARMv6 != "" {
+		imageCandidates = append(imageCandidates, ImageDef{Image: profile.ImageARMv6, Platform: "linux/arm/v6"})
+	}
+	if profile.Image386 != "" {
+		imageCandidates = append(imageCandidates, ImageDef{Image: profile.Image386, Platform: "linux/386"})
+	}
+	if profile.Image != "" {
```

Comment:
readme L83 says plain 'image' will override the platform-specific ones, but this looks like it will come last and presumably lose?

Reply:
So this is the update code, which isn't quite the same scenario as that readme line has in mind (where it's about selecting the image to be RUN.) So... I think it's okay as is.

To clarify: The key thing was that when we only had two architectures, it was assumed that if you set one architecture specific image, you were setting both (otherwise you'd just use the non-architecture "profile.Image") But now that we have a bunch, I don't think it's a safe assumption that people will always provide ALL architectures.

So, I figure it's better to just try to update ALL images that are set for a profile. As profile.Image itself _could_ potentially be set by CLI as well, I just append it too. So it's not that it will lose, it will just ALSO get downloaded. Which is nearly a no-op if it was already downloaded as one of the arch-specific targets above, so was just easier (and more concise) to do it this way than add more conditionals.

Oh, and note this should only be applying to manually triggered updates where people likely expect this. The automatic runtime updates (based on image download age) only hit for the actual image about to be run.